Sources/PaymentsCore/Networking/Enums/Environment.swift

```diff
@@ -0,0 +1,20 @@
+import Foundation
+
+public enum Environment {
+    case sandbox
+    case stage
+    case production
+
+    //swiftlint:disable force_unwrapping
+    var baseURL: URL {
+        switch self {
+        case .sandbox:
+            return URL(string: "https://api.sandbox.paypal.com")!
+        case .stage:
+            return URL(string: "https://api.msmaster.qa.paypal.com")!
```

Comment:
I don't think msmaster is accessible off of PayPal VPN so we may want to either omit this and add a confluence note somewhere if folks want to test on stage or add a comment above this noting that unless you're a PayPal employee this will be of no use to you. 😄

Reply:
ooohh this is a great point!!!
I wonder if it sufficient just testing on sandbox and remove stage completely considering our SDK being open source and this is public facing :-?

Reply:
Yeah, that's what I am more inclined to do as well! I think sandbox is typically sufficient for our testing purposes, but we could also add an internal confluence page for PayPal employees looking to test on msmaster with instructions on how to do that.

Reply:
+1 to this!